### ISSUE TITLE
Is rTorrent integration broken again? (API changes?)

### ISSUE THREAD

**Issue by JohanSF** [evilhero/mylar#2246]
![image](https://user-images.githubusercontent.com/6782603/55034789-a3419e80-5016-11e9-9608-331c48bd0389.png)
rTorrent version 0.9.6

**Comment by evilhero** [comment]
I run 0.9.6 and everything works for me. The recent api changes (months ago) were to allow for 0.9.7 api changes. 

Which branch /commit are you're running? Try turning on debug/verbose mode and trying a snatch again.

Also, try to follow the issue template when posting issues next time - it saves us having to ask branch/commit and debug logs everytime and speeds up responses usually.

**Comment by JohanSF** [comment]
Closing this issue. It turned out to be the error you get when the Post-Processing run meets a folder/file with illegal characters. I deleted the folder and now everything is fine.

I am on ":latest" of this container: https://hub.docker.com/r/linuxserver/mylar/tags
Version: c79cbe5a0eadd539801a5e3377fecebb852c1fe3

Sorry about not using the template, I'll do that next time - and thank you for the quick response evilhero.
